style.css

```diff
@@ -1,340 +1,656 @@
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    padding: 0 150px;
+}
+
+.team__item {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 316px;
+    height: 293px;
+    margin: 0 30px 30px 0;
+    padding: 30px;
+    background-color: #1D1C28;
+}
+
+.team__image {
+    display: block;
+    border-radius: 50%;
+}
+
+.team__name,
+.team__role {
+    text-align: center;
+}
+
+/* Index page About block */
+
+.about__wrapper {
+    max-width: 960px;
+    margin: 0 auto;
+}
+
+.about__title {
+    font-family: Bakbak One;
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 32px;
+    color: #fff;
+}
+
+.about__text {
+    font-family: Poppins;
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 28px;
+    color: #fff;
+}
+
+.about__features {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.about__item {
+   width: 46%;
+   margin-right: 30px;
+   margin-bottom: 30px;
+   padding: 30px;
+   background-color: #1A1925;
+   box-sizing: border-box;
+}
+
+.about__item:nth-child(even) {
+    margin-right: 0px;
+}
+
+.main--two-col {
+    max-width: 1280px;
+    margin: 0 auto;
+    display: flex;
+}
+
+.block__btns {
+    display: flex;
+}
+
+.btn__wrapper {
+    display: flex;
+    text-align: center;
+}
+
+.btn__wrapper--center {
+    justify-content: center;
+}
+
+.logo {
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    text-decoration-style: none;
+    text-decoration-line: none;
+}
+
+.logo__image {
+    margin-right: 15px;
+}
+
+.logo__text {
+    font-family: Bakbak One;
+    font-size: 32px;
+    font-weight: 400;
+    line-height: 42px;
+    text-align: left;
+    color: #ffffff;
+    text-decoration: none;
+    text-decoration-style: none;
+    text-decoration-line: none;
+    text-transform: uppercase;
+}
+
+.social {
+    display: flex;
+}
+
+li.social__item {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 40px;
+    height: 40px;
+    margin-right: 8px;
+    padding: 10px;
+}
+
+/* Index page FAQs block */
+
+.faq__items {
+    width: 960px;
+    margin: 0 auto;
+}
+
+.faq__item {
+    position: relative;
+    margin-bottom: 15px;
+    padding: 24px;
+    background-color: rgba(29, 28, 40, 1);
+}
+
+.faq__item[open] {
+    animation-name: fadeInDown;
+    animation-duration: 0.5s;
+}
+
+.faq__item--answer {
+    margin-top: 10px;
+    padding: 10px 24px 0px;
+    border-top: 1px solid rgba(191, 197, 238, 0.12);
+}
+
+details>summary {
+    list-style: none;
+}
+
+summary::-webkit-details-marker {
+    display: none
+}
+
+summary::after {
+    position: absolute;
+    content: '+';
+    right: 24px;
+}
+
+details[open] summary:after {
+    position: absolute;
+    content: "-";
+    right: 24px;
+}
+
+/* Footer */
+
+footer.footer {
+    background-color: #1A1925;
+}
+
+.footer__main {
+    display: flex;
+    justify-content: space-between;
+    max-width: 1600px;
+    margin: 0 auto;
```

Comment:
add padding left and right 15px and move bottom border on ::after

Reply:
done